bootstrap-salt.sh

```diff
@@ -2783,7 +2805,7 @@ install_debian_git_post() {
             if [ -f "${_SALT_GIT_CHECKOUT_DIR}/debian/salt-$fname.init" ]; then
                 __copyfile "${_SALT_GIT_CHECKOUT_DIR}/debian/salt-$fname.init" "/etc/init.d/salt-$fname"
             else
-                __fetch_url "/etc/init.d/salt-$fname" "http://anonscm.debian.org/cgit/pkg-salt/salt.git/plain/debian/salt-${fname}.init"
+                __fetch_url "/etc/init.d/salt-$fname" "https://anonscm.debian.org/cgit/pkg-salt/salt.git/plain/debian/salt-${fname}.init"
```

Comment:
This is definitely should be written with https:// from the start.
But in general, I don't think that downloading those init files from elsewhere is a good idea.

It's `install_debian_git_post()` function, so System V style initscripts just need to be included in the main Git repo.
As for now, the initscripts which required for Git installation on Debian 7 are located in [salt-pack](https://github.com/saltstack/salt-pack) repository.